torch/nn/quantized/modules/conv.py

```diff
@@ -38,14 +39,27 @@ def __init__(self, weight, bias, scale, zero_point, dtype,
         self.groups = groups
         self.padding_mode = padding_mode
 
-        self.weight = weight
-        self._packed_weight = ops.quantized.fbgemm_conv_prepack(weight, groups)
-        self.bias = bias  # torch.nn.Parameter(bias, requires_grad=False)
+        self._packed_weight = ops.quantized.fbgemm_conv_prepack(weight,
+                                                                self.groups)
+        self.bias = bias
+        self.bias.requires_grad = False  # Inference only!
```

Comment:
I think users will notice when they run backward? I feel just change bias to requires_grad=False does not prevent them to run backward since they will still hit some other tensor that don't have this set to False?

Reply:
Yea, this was a temp hack. Cleaned it up :)